index.html

```diff
@@ -24,22 +25,6 @@
                title="Bazel"
                alt="Bazel logo"
                class="logo img-responsive" />
-          <p class="tagline">{Fast, Correct} - Choose two</p>
```

Comment:
We reached internal agreement on the content of the landing page in a Doc that I sent you for review. If there were concerns, a meeting would be much more efficient than PR comments. Why is this only coming up now?

A few points:

1:
"<fast, correct> chose two" is a tagline. This is basically the Wolkswagen's "Das Auto", McDonalds' "I'm loving it". These brands use their tagline at the end of their TV ads below their logos, not at the beginning.
A tagline differs from what the users should read first when opening our page.
What the user should read is a concise description of the product conveying the key value proposition. This is the methodology that is used on [every product pages of GCP](https://cloud.google.com/products/) (click on any of these products, read below the title)

2:
My proposal was *not* to get rid of the Bazel tagline. I suggest to keep the tagline, but keeping is closer to the logo, closer to our brand.

3:
I am fully open to iterate on the description. A GitHub PR is not the best medium to do so. Let's use Google Docs.

4:
Our designers recently suggested us to use https://www.firebase.com as a reference, as it is for them a perfect example of landing page.
As you can see, they have this massive "Firebase helps you build better mobile apps and grow your business." at the top. They do not have any tagline.